.github/workflows/e2e-test-weekly.yml

```diff
@@ -370,7 +370,7 @@ jobs:
       scheduled: ${{ github.event_name == 'schedule' }}
 
   e2e-tf-module:
```

Comment:
I think we should delete this job, as the test won't work anymore since we don't publish the module anymore.

Reply:
this should be reworked for the provider

Reply:
What part of it would be broken? I just renamed the workflow in preparation of the provider adjustment.

Reply:
With this PR, we only upload the infrastructure module, for which the given inputs are not sufficient (inputs of the legacy whole-cluster module and the infrastructure module are not the same). Additionally, This test makes no sense in its current form, as it only creates the infrastructure but doesn't test anything on our side. My main point is it being broken due to the inputs though.